### PR TITLE
AV-212390 Reverting BackendRefFilter changes

### PR DESCRIPTION
**Pull request by pkoshtavmware**
AV-212390 Reverting BackendRefFilter changes

### CODE REVIEW

**Comment by pkoshtavmware** [comment]
UT runs:
`ok      github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tests/dedicatedevhtests     692.774s        coverage: 28.2% of statements in ./...`

**Review by DixitAakash — COMMENTED** [review]
I have only one small comment. The PR otherwise looks good. I have also run all the regex and app-root FT's and all are running fine.

File: ako-gateway-api/nodes/avi_model_l7_translator.go [hunk 1]
```diff
@@ -227,7 +227,6 @@ func (o *AviObjectGraph) BuildPGPool(key, parentNsName string, childVsNode *node
 		}
 
 		// TODO: Check Backend filter code. This is creating an issue in checksum calculation of object which is result in failure in deletion
```
[the remark this answers is not in the document]
Comment (pkoshtavmware) [reply ako-gateway-api/nodes/avi_model_l7_translator.go]: Done

**Review by DixitAakash — DISMISSED** [review]
LGTM

**Comment by DixitAakash** [comment]
build ako

**Review by DixitAakash — APPROVED** [review]
LGTM

**Review by arihantg — APPROVED** [review]
LGTM

**Comment by pkoshtavmware** [comment]
build ako

**Pull request merged**